creator-node/src/routes/users.js

```diff
@@ -28,8 +28,16 @@ module.exports = function (app) {
     return successResponse()
   }))
 
+  app.get('/users/login/challenge', handleResponse(async (req, res, next) => {
+    const walletPublicKey = req.query.walletPublicKey
+    const redisClient = req.app.get('redisClient')
+    const challenge = (new Date()).getTime()
+    redisClient.set(walletPublicKey, challenge)
+    return successResponse({ walletPublicKey, challenge })
+  }))
+
   app.post('/users/login', handleResponse(async (req, res, next) => {
-    const { signature, data } = req.body
+    const { signature, data, challenge } = req.body
```

Comment:
@vicky-g this route needs to be backwards compatible (maybe add new branching logic/new route for new auth)

Reply:
added `/users/login/challenge` to handle new challenge logic